lib/io/console/jruby/native_console.rb

```diff
@@ -0,0 +1,153 @@
+# Load appropriate native bits for BSD or Linux
+case RbConfig::CONFIG['host_os'].downcase
+when /darwin|openbsd|freebsd|netbsd/
+  require 'io/console/bsd_console'
```

Comment:
Isn't it 'io/console/jruby/bsd_console'?

Reply:
You are correct...I didn't realize these paths were used in the specific impls too. I will correct and re-push.